Lib/test/test_tempfile.py

```diff
@@ -957,6 +957,13 @@ def test_bad_mode(self):
             tempfile.NamedTemporaryFile(mode=2, dir=dir)
         self.assertEqual(os.listdir(dir), [])
 
+    def test_if_temp_file_already_removed(self):
+        tmpdir = tempfile.mkdtemp()
+        self.addCleanup(support.rmtree, tmpdir)
+        with tempfile.NamedTemporaryFile(delete=True, dir=tmpdir) as fp:
+            os.system('rm {}'.format(fp.name))
```

Comment:
Why does this use os.system to call rm, rather than calling os.unlink? 'rm' is not exactly cross-platform. This test will also fail on Windows (and other systems where you can't delete files that are currently in use). It'll have to be skipped on those platforms.